README.md: Refactor markdown, cleanup links (#368)
voxpupuli/hiera-eyaml: README.md

#### RubyGems

    $ gem install hiera-eyaml

#### Apt (Ubuntu 18.04+)

    $ sudo apt install hiera-eyaml

### Installing hiera-eyaml for [puppetserver](https://github.com/puppetlabs/puppetserver)

> All commands need to be executed as root. Puppet Enterprise vendores hiera-eyaml

Reviewer: ```suggestion
All commands need to be executed as root. Puppet Enterprise vendors hiera-eyaml
```

already, so you don't need to install it there.

```sh
puppetserver gem install hiera-eyaml
```

or via puppet:

```sh
puppet resource package hiera-eyaml ensure=installed provider=puppetserver_gem
```

or via Puppet DSL:

> ```

Reviewer: ```suggestion
```puppet
```

If you have found a bug then please raise an issue here on github.

> Some of us hang out on #voxpupli on Libera.Chat, please drop by if you want to say hi or have a question.

Reviewer: ```suggestion
Some of us hang out on #voxpupuli on [Libera.Chat](https://libera.chat/), please drop by if you want to say hi or have a question.
```
